packages/plugin-compiler/src/main/kotlin/io/realm/compiler/RealmSchemaLoweringExtension.kt

```diff
@@ -153,15 +153,15 @@ class RealmSchemaLoweringExtension : IrGenerationExtension {
  *
  * E.g. all of these are valid
  * ```
- * RealmConfiguration(schema = setOf(MyType::class))
```

Comment:
It feels a bit of a shame to completely removing this. Having to always go through the Builder is quite a lot of overhead.

In Java we have a factory constructor like `SyncConfiguration.defaultConfig(....)` 

Doing something similar here would seem nice?

Reply:
Yeah. Probably nice to keep something like that.

Reply:
Added `RealmConfiguration.defaultConfig(...)`. Implementation just forwards any arguments to the builder constructor `RealmConfiguration.Builder(...)` and returns the built configuration.